static/app/components/compactSelect/list.tsx

```diff
@@ -1,26 +1,27 @@
-import {Fragment, useContext, useEffect, useMemo, useRef, useState} from 'react';
```

Comment:
This file used to be `ListBox`. Now it's split into 2 parts: a `List` component (here) that initializes the list state (via `useListState()`), and a `ListBox` component (see `'./listBox'`) that only handles accessibility features (via `useListBox()`). 

This is so we don't have duplicate code in `GridList`. Both `ListBox` and `GridList` consume the same list state object, so it's better to initialize that state here and pass it to either component for further use.